source/docs/en/guide_github.md

```diff
@@ -1,31 +1,60 @@
 # Using GitHub
-If you want to work on codecheck challenges in your local environment, you can!  
-First you'll need a [github account](https://github.com/join).  
-Then, [connect your GitHub account](https://app.code-check.io/settings/social) to codecheck before you start the challenge.
+Unlike other platforms, if you want to work on codecheck challenges in your local environment,  
+you can!  
 
-## Solving with GitHub
-### Fork challenge
-This menu appears when you click the "Fork" button.
-![イメージ11](images/s11.png)  
-Fill out the repository name and click the "OK" button, then you are going to jump to the generated repository page.  
-Check "Private Repository" if you do not want to publish the repository.
+## Requirements
+- <a href="https://github.com/join" target="_blank"> GitHub account</a>
+- <a href="https://git-scm.com/" target="_blank">Git</a> (If you don't know Git, see <a href="https://try.github.io/" target="_blank">this</a> and <a href="https://git-scm.com/doc" target="_blank">this</a>.)
+- <a href="https://nodejs.org/en/download/" target="_blank">Node.js/npm</a>
+- <a href=
+"https://app.code-check.io/settings/social" target="_blank">Connect your GitHub account</a> to code-check.io.
+- Finally, chant `npm install codecheck` in the CLI to summon codecheck.
 
-### Clone repository
-![イメージ15](images/s15.png)   
-Clone your challenge repo with the standard `git clone`:
+<!-- - First you'll need a [GitHub account](https://github.com/join).
+- And of course [Git](https://git-scm.com/).(If you don't know git, see [this](https://try.github.io/) and [this](https://git-scm.com/doc).)  
+- Then, [connect your GitHub account](https://app.code-check.io/settings/social) to code-check.io.
+- Also, get [nodejs/npm](https://nodejs.org/en/download/)
+- And finally, chant `npm install codecheck` in the CLI to summon codecheck. -->
 
-```
-$ git clone {GIT_URL}
-```
-Now, you can hack Challenges in your own local environment!
+## Setup
+
+![Setting up your challenge locally with GitHub](images/start_challenge_github.gif)
 
-### Save answer
-Save your answer by pushing your branch to remote master.
+- **Complete all requirements**.
+- Click the ![Fork to / Open with GitHub](images/open_github.png) Button.
+- Fill out the repository name and click the "OK" button.
+- Click the link to jump to the generated GitHub repository page.
+- Copy the repo link from the "Clone or Download" menu.
+- Go to your CLI and `git clone {REPO NAME}` to copy the repository to your local directory.
+- Now, just launch your favorite IDE/editor and crush that challenge!
 
+## Running the Tests
+
+- Once you're ready to test your code, run `codecheck` from your CLI.
+- The test results will print to stdout like this:
 ```
-$ git push origin master
+$ codecheck
```

Comment:
Thanks for the good suggestion!
Unfortunately, I just tried this and I get inconsistent results with syntax coloring, probably because most of the syntax are not standard bash commands. :(
Thus, I think we should stick with the current syntax...